docs/versions/content-patterns.md

```diff
@@ -0,0 +1,317 @@
+
+**When to use:** When the section-level scope differs from the page-level scope
+
+**Example**
+See above
+
+## Inline `applies` tags
+
+::::{warning}
+This feature doesn't currently work - we're working around it by using prose.
+::::
+
+**Use case:** When features in a **list of features** are exclusive to a specific context, or were introduced in a specific version
+
+**Number to use:** Three max. If the number of tags is longer than that, consider: 
+
+*  Breaking up the list by facet
+*  Using labels that don't have version / lifecycle information and deferring that information to the page or section for the feature
+  
+Currently, we don't have inline `applies` tags. Instead, append the information to the bullet item in bolded square brackets `**[]**`. Add all information within a single set of square brackets. The brackets should appear after any final punctuation. 
+
+Because this approach is less clear, consider using words like `only` to help people to understand that this information indicates the applicability of a feature.
+
+### Example
+
+Spaces let you organize your content and users according to your needs.
+
+::::{tab-set}
+:group: one-two
+
+:::{tab-item} One
+:sync: one
+
+* Each space has its own saved objects.
+* Users can only access the spaces that they have been granted access to. This access is based on user roles, and a given role can have different permissions per space.
+* Each space has its own navigation. **[Stack v9 only]**
+
+:::
+:::{tab-item} Two
+:sync: two
+
+
+* Learn about internal users, which are responsible for the operations that take place inside an Elasticsearch cluster
+* Learn about service accounts, which are used for integration with external services that connect to Elasticsearch
+* Learn about the services used for token-based authentication
+* Learn about the services used by orchestrators **[Elastic Cloud Hosted, Elastic Cloud Enterprise, Elastic Cloud on Kubernetes]**
+* Learn about user lookup technologies
+* Manage the user cache
+
+:::
+::::
+
+
+## Tabs
+
+**Use case:** When one or more steps in a process differs, put the steps into tabs - one for each process.
+
+**Number to use:** Max 4 or 5 (in a deployment type / versioning context - might be different for other situations)
+
+Try to minimize the number of tabs where you can - try to work around small differences by rewording or adding context in prose or in `note` style admonitions:
+
+* In version 9.1 and earlier, **Spaces** were referred to as **Places**. 
+* 
+  ::::{note}
+  In version 9.1 and earlier, click the **Permissions** tab.
+  ::::
+
+Try not to include information surrounding a procedure in the tabs. Make the tab content as small as possible apart from the procedure itself.
+
+Consider breaking up procedures into sets of procedures if only one section differs between contexts.
+
+### Example
+
+To create a space: 
+
+:::::{tab-set}
+:group: stack-serverless
+
+::::{tab-item} {{serverless-tab}}
```

Comment:
> ::::{tab-item} {{serverless-tab}}

I pushed a commit to use substitutions for the tab labels, which should hopefully work per https://github.com/elastic/docs-builder/issues/428